(GH-153) Ensure generated readme contains module name info (#163)
puppetlabs/Puppet.Dsc: CHANGELOG.md

- Ability to set generated modules fixtures to something other than the latest released version of `puppetlabs-pwshlib` on the Forge ([#93](https://github.com/puppetlabs/Puppet.Dsc/issues/93))
- A check to `New-PuppetDscModule` which validates that PSRemoting is enabled and errors clearly if not in order to prevent unexpected execution failures later in the process ([#133](https://github.com/puppetlabs/Puppet.Dsc/issues/133))

### Fixed

> - Have ensured that the auto generated readme contains both the PowerShell and Puppet module names ([#153](https://github.com/puppetlabs/Puppet.Dsc/issues/153))

Reviewer: ```suggestion
- Ensure that the auto-generated readme of a Puppetized module contains both the PowerShell and Puppet module names ([#153](https://github.com/puppetlabs/Puppet.Dsc/issues/153))
```

> - With the exception of `PuppetModuleName` all `Update-PuppetModuleReadme` parameters have been set to mandatory and to not except null or empty values. `PuppetModuleName` has been left as when not given it will simply mirror the `PowerShellModuleName` ([#153](https://github.com/puppetlabs/Puppet.Dsc/issues/153))

Reviewer: ```suggestion
- Make all `Update-PuppetModuleReadme` parameters except `PuppetModuleName` mandatory and do not accept null or empty values ([#153](https://github.com/puppetlabs/Puppet.Dsc/issues/153))
```

> - For `Get-Readmecontent` the parameters for `PowerShellModuleName` and `PuppetModuleName` have both been set to mandatory and all values have been set to not accept null or empty values ([#153](https://github.com/puppetlabs/Puppet.Dsc/issues/153))

Reviewer: ```suggestion
- Make the  `PowerShellModuleName` and `PuppetModuleName` parameters in `Get-Readmecontent` mandatory and do not accept null or empty values for any parameters ([#153](https://github.com/puppetlabs/Puppet.Dsc/issues/153))
```
